From 526a0409f21e96b5c4e1b9a469c22f7b00c9e2d8 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Fri, 27 Aug 2021 14:10:18 +0530
Subject: [PATCH] Revert "VTS fix for Thermal"

This reverts commit aa418b0144349f98825cf9ef4d31ef242899714e.

In android 12, this patch was giving boot issues as the
android.hardware.thermal@1.0 superclass of
android.hardware.thermal@2.0 not in manifest.
---
 groups/device-specific/caas/framework_manifest.xml | 1 +
 groups/device-specific/caas/manifest.xml           | 1 +
 2 files changed, 2 insertions(+)

diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index f17bea2..f14100d 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -34,6 +34,7 @@
     <hal format="hidl">
         <name>android.hardware.thermal</name>
         <transport>hwbinder</transport>
+        <version>1.0</version>
         <version>2.0</version>
         <interface>
             <name>IThermal</name>
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 46185ca..6da19df 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -34,6 +34,7 @@
     <hal format="hidl">
         <name>android.hardware.thermal</name>
         <transport>hwbinder</transport>
+        <version>1.0</version>
         <version>2.0</version>
         <interface>
             <name>IThermal</name>
-- 
2.17.1

